api/pom.xml

```diff
@@ -56,10 +56,13 @@
 			<groupId>com.google.guava</groupId>
 			<artifactId>guava</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>com.github.java-json-tools</groupId>
+			<artifactId>json-patch</artifactId>
+		</dependency>
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
-			<scope>test</scope>
```

Comment:
So seeing this now, it looks like *I* was the one that moved this out of test scope, because I'm using Jackson in the JsonPatchUtils file... does this seem okay @ibacher ?

Reply:
(Although this looks like it already was included in the omod pom.xml as a normal dependency, so I think it would have been packaged with the omod irregardless (and necessary for the 2.1 and below Core versions, like you said?)

Reply:
Yeah... I think it's perfectly fine not to have this `test` scope. This is probably a result of the time I tried to cut down the size of the OMOD as much as possible (embedding the HAPI code and—even worse—the FHIR Jars makes this module relatively large).